src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IParameterReferenceExpression.vb

```diff
@@ -238,33 +238,48 @@ Friend Class [Class]
     Public Property C As [Class]
 
     Public Sub M(x As Integer, y As Integer, z As Integer)
-        Dim c = New [Class]() With {'BIND:"New [Class]() With {"
+        Dim c = New [Class]() With {'BIND:"New [Class]() With {"'BIND:"New [Class]() With {'BIND:"New [Class]() With {""
```

Comment:
Seems like they got left behind accidentally when I copied another existing test and modified it. There are few more tests with extra bind comments, I will clean them separately.